app/src/test/java/com/github/multimatum_team/multimatum/AddDeadlineTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.github.multimatum_team.multimatum
+
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.contrib.PickerActions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.multimatum_team.multimatum.repository.DeadlineRepository
+import com.github.multimatum_team.multimatum.util.MockDeadlineRepository
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import dagger.hilt.android.testing.UninstallModules
+import dagger.hilt.components.SingletonComponent
+import org.hamcrest.CoreMatchers
+import org.hamcrest.MatcherAssert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.shadows.ShadowToast
+import javax.inject.Singleton
+
+
+/**
+ * Class test for AddDeadlineActivity
+ */
+@UninstallModules(RepositoryModule::class)
+@HiltAndroidTest
+@RunWith(AndroidJUnit4::class)
+class AddDeadlineTest {
+
+    @get:Rule
+    var hiltRule = HiltAndroidRule(this)
```

Comment:
The Hilt Rule should always be executed first. From what I understood, the rules (i.e. the things annotated with @Rule) execute in the order they are declared, so this should not be an issue for now. However, it would be safer to enforce the ordering:
```Kotlin
@get:Rule(order = 0)
    var hiltRule = HiltAndroidRule(this)
```
And then orders 1, 2, etc. for the following rules.